setup.cfg

```diff
@@ -3,5 +3,4 @@ zip_ok = False
 
 [pep8]
 max-line-length=999
-ignore=E121,E123,E126,E226,E24,E704,E221,E127,E128,W503,E731,E131,E711,E712
-exclude=test_*
\ No newline at end of file
+ignore=E121,E123,E126,E226,E24,E704,E221,E127,E128,W503,E731,E131,E711,E712,E402
```

Comment:
Two things:

1) ``ignore=test_*`` was a holdover from when our unit tests were totally broken and that got copy/pasted into this repo.  It shouldn't be there, because we want/expect our unit tests to pass PEP8 validation as well.

2) ``E403`` is a convention we explicitly disregard; it says that all module-level imports should come at the top of a file.